nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/status/S3BinaryManagerStatusProbe.java

```diff
@@ -30,22 +28,24 @@
 
 /**
  * Probe to check the status of the S3BinaryManager. Returns success if Nuxeo can access the bucket, failure otherwise
- * 
+ *
  * @since 9.3
  */
 public class S3BinaryManagerStatusProbe implements Probe {
 
     @Override
     public ProbeStatus run() {
-
-        Map<String, BlobProvider> providers = Framework.getService(BlobManager.class).getBlobProviders();
-        for (BlobProvider blobProvider : providers.values()) {
+        BlobManager blobManager = Framework.getService(BlobManager.class);
+        for (BlobProvider blobProvider : blobManager.getBlobProviders().values()) {
```

Comment:
If you prefer you could have done this will a stream `filter` `anyMatch`

Reply:
Some other time :)